packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -456,6 +470,11 @@ class _ExpansionPanelListState extends State<ExpansionPanelList> {
 
   @override
   Widget build(BuildContext context) {
+    assert(kElevationToShadow.containsKey(widget.elevation) != false,
+      'Invalid value for elevation. Only the following values can be used to define'
+      ' the elevation: 0, 1, 2, 3, 4, 6, 8, 9, 12, 16, 24'
```

Comment:
I would avoid using the full set of values here. Maybe do a pointer to api.flutter.dev/flutter/material/kElevationToShadow-constant.html instead, since the problem with this would be any change to the values in kElevationToShadow might cause this to accidentally be off. For example, if we add an elevation value for 48 and forget to update this assert message, it'll be incorrect.


```suggestion
      'Invalid value for elevation. See the kElevationToShadow '
      'constant for possible elevation values.'
```

Reply:
Makes sense, I'll do the changes.